src/segmented-control/segmented-control.scss

```diff
@@ -0,0 +1,160 @@
+// SegmentedControl
+
+.SegmentedControl {
+  display: inline-flex;
+  background-color: var(--color-segmented-control-bg);
+  // stylelint-disable-next-line primer/borders
+  border-radius: var(--primer-borderRadius-medium, $border-radius);
+  // stylelint-disable-next-line primer/box-shadow
+  box-shadow: var(--primer-borderInset-thin, inset 0 0 0 $border-width) var(--color-border-default);
+}
+
+// Button -----------------------------------------
+
+.SegmentedControl-button {
+  position: relative;
+  display: inline-flex;
+  height: var(--primer-control-medium-size, 32px);
+  // stylelint-disable-next-line primer/spacing
+  padding: 0 var(--primer-control-medium-paddingInline-normal, 12px);
+  // stylelint-disable-next-line primer/typography
+  font-size: var(--primer-text-body-size-medium, $body-font-size);
+  color: var(--color-fg-default);
+  background-color: transparent;
+  // stylelint-disable-next-line primer/borders
+  border: var(--primer-borderWidth-thin, $border-width) $border-style transparent;
+  // stylelint-disable-next-line primer/borders
+  border-radius: var(--primer-borderRadius-medium, $border-radius);
+  transition: border-color, background-color 180ms cubic-bezier(0.3, 0.1, 0.5, 1);
+  align-items: center;
+  justify-content: center;
+  gap: var(--primer-control-medium-gap, $spacer-2);
+
+  &:hover {
+    background-color: var(--color-segmented-control-button-hover-bg);
```

Comment:
Interesting.. it feels a bit too much inset. But we would not need to hide the dividers when hovering.

Reply:
I briefly explored that hover state– wondering if it might make sense to go lighter, as to imply it will be selected (the lightest shade) on click

![image](https://user-images.githubusercontent.com/18661030/172874262-ee7851af-1a79-4273-b8c4-a1bee66634d5.png)

Reply:
I think in our sync with @vdepizzol we decided light didn't make sense here 😄 so disregard my suggestion! I agree Mike we should try and stick with standard conventions.

Reply:
Thanks @langermank - I completely forgot about the discussion in our sync 😅

Reply:
Ok, it now uses a `3px` inset padding:

![2022-07-20 16 28 58](https://user-images.githubusercontent.com/378023/179923006-c01c40fc-d45b-45b5-8fc5-282111c7b222.gif)